pages/api/sign-up.js

```diff
@@ -46,7 +46,10 @@ async function handler(req, res) {
 
     // attempt to send validation email through notify
     try {
-      const validationUrl = origin + `/api/validate?id=${userCuid}`;
+      let lang = "";
+      data.language === "en" ? (lang = "") : (lang = "fr");
+      const validationUrl =
+        origin + `/api/validate?id=${userCuid}&lang=${lang}`;
```

Comment:
> const validationUrl = origin + `/api/validate?id=${userCuid}&lang=${lang}`;

You can just use `data.language` where you have `lang=${lang}` since the way it is now will just be `lang=` if a user submits the form in english.